docs/docs/workflows/terraform-module.md

```diff
@@ -0,0 +1,130 @@
+# Use the Terraform module
+You can manage a Constellation cluster through Terraform.
+The necessary module package is available as part of the release and consists of a convenience module for each cloud service provider (`{csp}-constellation`) that combines the IAM (`infrastructure/{csp}/iam`), infrastructure (`infrastructure/{csp}`), and constellation (`constellation-cluster`) modules.
+
+## Prerequisites
+- a Linux / Mac operating system
+- a Terraform installation of at least `v1.4.4`
+
+## Quick setup
+The convenience module allows setting up a Constellation cluster with a single module. It's easiest to consume the module through a remote source, as shown below.
+This allows to upgrade the cluster to a newer Constellation version by simply updating the module source.
+
+1. Create a directory (workspace) for your Constellation cluster.
+  ```bash
+  mkdir c11n
+  cd c11n
+  ```
+
+1. Create a `main.tf` file to call the CSP specific Constellation module.
+
+  <tabs groupId="csp">
+
+  <tabItem value="aws" label="AWS">
+
+  ```
+  module "aws-constellation" {
+    source = "https://github.com/edgelesssys/constellation/releases/download/$VERSION/terraform-module.zip//modules/aws-constellation" # TODO check the link (name the to be zipped dir modules) during the release. So far only tested with https://tfmodule.s3.eu-west-1.amazonaws.com/modulesV2.zip//modules/aws-constellation
+    name        = "aws-example"
+    image       = "$VERSION" // replace with the desired version, e.g., `v2.13.0`.
```

Comment:
URL also contains $VERSION. And this is easily overlooked.

Reply:
remove image here, as there now is a default: https://github.com/edgelesssys/constellation/pull/2566/commits/d1debdec7da9277af7f34d42ff9f898d1db143b8